app/models/miq_product_features_share.rb

```diff
@@ -0,0 +1,2 @@
+class MiqProductFeaturesShare < ApplicationRecord
```

Comment:
What's the table called? If we're creating a model I would follow the rules for model to table mapping.

For example, the `cloud_tenants_vms` table does not have a model file. So that follows the convention for join tables (both pluralized and in lexicographical order). And if we were to create a model I would expect it to be called `CloudTenantsVm` so that we could use `.table_name` properly.

On the other side of things there is the `host_storages` table which has a model called `HostStorage` (which just defines `belongs_to :host` and `belongs_to :storage`.

So if nothing else, I would just make sure `.table_name` works and we'll be fine :smile: